### PR TITLE
Update website plugins documentation

### PR DESCRIPTION
**Pull request by alepuccetti**
This PR adds information about plugins useful to developers who want create their own plugins, explaining the basic structure of a plugin, report and control interfaces.
Also, it adds a section about the currently implemented plugins with brief descriptions and links to the Weaveworks Plugins organization and the plugins repositories.

/cc @abuehrle can you give me some feedbacks on this first iteration about plugins documentation? I will add more information in later PRs.

/cc @2opremio

### CODE REVIEW

**Comment by abuehrle** [comment]
@alepuccetti I don't seem to be able to access your branch.  Can you add me to your repo and I will edit these for you.

File: site/plugins.md [hunk 1]
```diff
@@ -13,17 +14,21 @@ With a Scope probe plugin, you can insert custom metrics into Scope and have the
 
 ![Custom Metrics With Plugins](images/plugin-features.png)
 
-You can find some examples in [the example plugins](https://github.com/weaveworks/scope/tree/master/examples/plugins) directory.
+## <a id="official-plugins"></a>Official Plugins
 
-There are currently two different examples:
+You can find all the official plugins at [Weaveworks Plugins](https://github.com/weaveworks-plugins).
```
Comment (abuehrle) [inline site/plugins.md]: Official plugins can be found at [Weaveworks Plugins](https://github.com/weaveworks-plugins)

**Comment by alepuccetti** [comment]
@abuehrle I pushed the same branch on my [personal repo](https://github.com/alepuccetti/scope/tree/alessandro/site-plugins) and sent you an invite to collaborate on that. This should be enough to allow you to make changes, I will push your changes here later.

Sorry for the inconvenient but I cannot change the permission on the Kinvolk repo.

**Comment by 2opremio** [comment]
LGTM, but I would like @abuehrle to also sign-off on this one before merging. @abuehrle Are you OK with this PR?

**Comment by abuehrle** [comment]
LGTM

**Comment by 2opremio** [comment]
docs republished: https://www.weave.works/documentation/scope-latest-plugins/

**Pull request merged**